src/redux/reducers/profile.js

```diff
@@ -0,0 +1,35 @@
+import { ADD_POST, SET_POST } from '../types/profile';
+
+const initialState = {
+  posts: [
+    {id: 1, postText: 'Hi, Max'},
+    {id: 2, postText: 'Hi, Eugene'},
+    {id: 3, postText: 'Hi, Yuri'},
+    {id: 4, postText: 'Hi, Alexei'},
+    {id: 5, postText: 'Hi, Andrey'}
+  ],
+  postText: 'Post text'
+};
+
+export const profileReducer = (state = initialState, action) => {
```

Comment:
Maybe call it like "postsReducer"? =)

Reply:
No, that is right!
"Posts" are part of the "Profile" and profileReducer.
In the future "Profile" and profileReducer will also contain "Info".